### PR TITLE
Release 2.22 Screenshots

### PR DESCRIPTION
**Pull request by larswmh**
chrome_G3rDI4EDwX
chrome_rVQdo2xVfq

New section: Rapid Test Profiles
image
chrome_RdPNzfFnXY

image
image

### CODE REVIEW

File: src/data/screenshots-archive-2-21.json [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
+                ]
+            },
+            {
+                "title": "Statistics",
+                "anchor": "android_stats",
+                "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```
Comment (MikeMcC399) [inline src/data/screenshots-archive-2-21.json:95]: doeses => doses

File: src/data/screenshots-archive-2-21.json [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_011_myQRCodes", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_008_traceLocationDetailScreen", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-traceLocation_009_traceLocationPdfScreen", "alt": "Check In image 8 of 8" }
+                ]
+            },
+            {
+                "title": "Statistics",
+                "anchor": "ios_stats",
+                "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```
Comment (MikeMcC399) [inline src/data/screenshots-archive-2-21.json:364]: doeses => doses

File: src/data/screenshots-archive-2-21.json [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-statistics_info_screen_0001", "alt": "Statistics image 12 of 13" },
+                    { "filename": "/assets/screenshots/2-21/en/ios/iPhone 13-statistics_info_screen_0002", "alt": "Statistics image 13 of 13" }
+                ]
+            },
+            {
+                "title": "Test Registration & Key Sharing",
+                "anchor": "ios_registertest",
+                "description": "In this section you will find screenshots of test registration and key sharing. If you press \"Continue\" on the \"Status\" page at \"You are getting tested\" you will get to the test administration. Here you have the possibility to register your PCR test or the PCR test of a famlily member. Press \"Enter TAN for PCR test\" so that a window appears where you can enter your 10-digit TAN. Once you have done this, you still need to give your consent. Then your test result can have three different statuses. These are \"Your result is not yet available\" \"SARS-CoV-2 negative,\" and \"SARS-CoV-2 positive.\" With the latter, you still have the option to record the onset and type of symptoms.",
```
Comment (MikeMcC399) [inline src/data/screenshots-archive-2-21.json:384]: famlily => family

File: src/data/screenshots.json [hunk 4]
```diff
@@ -30,250 +30,250 @@
...
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
                 ]
             },
             {
                 "title": "Statistics",
                 "anchor": "android_stats",
                 "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:95]: doeses => doses

File: src/data/screenshots.json [hunk 5]
```diff
@@ -286,257 +286,257 @@
...
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_011_myQRCodes", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_008_traceLocationDetailScreen", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-traceLocation_009_traceLocationPdfScreen", "alt": "Check In image 8 of 8" }
                 ]
             },
             {
                 "title": "Statistics",
                 "anchor": "ios_stats",
                 "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:364]: doeses => doses

File: src/data/screenshots.json [hunk 6]
```diff
@@ -286,257 +286,257 @@
...
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_info_screen_0001", "alt": "Statistics image 12 of 13" },
+                    { "filename": "/assets/screenshots/2-22/en/ios/iPhone 13-statistics_info_screen_0002", "alt": "Statistics image 13 of 13" }
                 ]
             },
             {
                 "title": "Test Registration & Key Sharing",
                 "anchor": "ios_registertest",
                 "description": "In this section you will find screenshots of test registration and key sharing. If you press \"Continue\" on the \"Status\" page at \"You are getting tested\" you will get to the test administration. Here you have the possibility to register your PCR test or the PCR test of a famlily member. Press \"Enter TAN for PCR test\" so that a window appears where you can enter your 10-digit TAN. Once you have done this, you still need to give your consent. Then your test result can have three different statuses. These are \"Your result is not yet available\" \"SARS-CoV-2 negative,\" and \"SARS-CoV-2 positive.\" With the latter, you still have the option to record the onset and type of symptoms.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:384]: famlily => family

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
- Typos corrections from PR https://github.com/corona-warn-app/cwa-website/pull/2810 also marked up here in-line.

File: src/data/screenshots.json (hunk 6 above)
Comment (larswmh) [reply src/data/screenshots.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!

File: src/data/screenshots.json (hunk 5 above)
Comment (larswmh) [reply src/data/screenshots.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!

File: src/data/screenshots.json (hunk 4 above)
Comment (larswmh) [reply src/data/screenshots.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!

File: src/data/screenshots-archive-2-21.json (hunk 3 above)
Comment (larswmh) [reply src/data/screenshots-archive-2-21.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!

**Review by dsarkar — APPROVED** [review]

**Pull request merged**